RyuSocks/Packets/Auth/Constants.cs

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (C) RyuSOCKS
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <https://www.gnu.org/licenses/>.
+ */
+
+namespace RyuSocks.Packets.Auth
+{
+    public static class Constants
+    {
+        public const byte UaPVersion = 0x01;
```

Comment:
```suggestion
        public const byte UsernameAndPasswordVersion = 0x01;
```

This is kind of ugly, but I'd like to avoid abbreviations in variable names.

Reply:
yes it is very ugly